### PR TITLE
[ZH/EN] Add Interface, update information in English article

### PR DESCRIPTION
**Pull request by cdwcgt**
Completed
*Note:This pr is not only translate but also perfection of the original.So need to check en.md.*

### CODE REVIEW

File: wiki/Interface/zh.md [hunk 1]
```diff
@@ -51,10 +57,260 @@ outdated: true
...
+
+这一部分叫做游玩工具箱。我们将会从左到右依次讲述各个按钮的功能。
+
+按下`Esc`或点击`返回`按钮返回到主界面。
+
+点击`Mode`按钮来打开可用的游戏模式列表。点击你想要游玩的游戏模式然后osu!会自动切换到那个模式的风格 - 排行榜也会自动切换，或者你可以使用`Ctrl`和`1`（osu!），`2`（osu!taiko），`3`（osu!catch），和`4`（osu!mania）来更换游戏模式。
+
+背景透明图标和“Mode”按钮会更改至当前选中的游戏模式的图标。
+
+![](img/Soloplay8-ZH.jpg "游戏修改器")
```
Comment (cdwcgt) [inline wiki/Interface/zh.md:238]:
```suggestion
![](img/Soloplay8-ZH.jpg "Game modifiers")
```

**Review by cdwcgt — COMMENTED** [review]
因为修改器实在别扭所以考虑要不要保留

File: wiki/Interface/zh.md [hunk 2]
```diff
@@ -2,46 +2,52 @@
...
-![](img/Useroptions1.png "User Options")
+![](img/Useroptions1-ZH.png "用户选项")
 
 点击屏幕左上角的用户面板以打开用户选项。此时将无法使用聊天界面。点击相应的选项以完成操作。
 
-1. **View Profile**：在浏览器中打开玩家主页
-2. **Sign Out**：退出登录
-3. **Change Avatar**：在浏览器中打开更改头像页面
-4. **Close** ：关闭此窗口
+1. `View Profile`：在浏览器中打开玩家主页。
```
Comment (kj415j45) [inline wiki/Interface/zh.md:39]:
```suggestion
1. `查看个人资料`：在浏览器中打开玩家主页。
```
已有本地化的字符串以游戏内显示为准，下同

File: wiki/Interface/zh.md [hunk 3]
```diff
@@ -2,46 +2,52 @@
...
-![](img/Intro_static.jpg "Introduction screen")
+![](img/Intro_static.jpg "介绍屏幕")
 
 ## 主菜单
 
-![](img/Introscreen.jpg "Main menu")
+![](img/Introscreen.jpg "主菜单")
 
-［1］osu！的标志会随着音乐的每分钟击打次数（BPM）跳动，如果没有在播放歌曲，osu！标志就会跳动得慢一点。下列在括弧里的是快捷键。
+［1］[osu！曲奇](/wiki/Glossary#cookie) \[1\][BPM](/wiki/Beatmapping/Beats_per_minute)跳动，此外，从osu！曲奇延伸出来的条的长度与歌曲的音量一致。如果没有在播放歌曲或者暂停播放，osu！曲奇就会以60BPM的速度跳动。主菜单的元素如下：
```
[the remark this answers is not in the document]
Comment (cdwcgt) [reply wiki/Interface/zh.md:13]: 那就粉饼吧

[the remark this answers is not in the document]
Comment (cdwcgt) [reply wiki/Interface/zh.md]: [1]后面已经有了所以就不改了(原文格式)

File: wiki/Interface/en.md [hunk 4]
```diff
@@ -154,7 +154,7 @@ Supported filters:
 - `cs`: Circle Size
 - `od`: Overall Difficulty
 - `hp`: HP Drain Rate
-- `keys`: Number of keys (osu!mania maps only)
+- `keys`: Number of keys (osu!mania maps and convered maps only)
```
Comment (TPGPL) [inline wiki/Interface/en.md:157]:
```suggestion
- `keys`: Number of keys (osu!mania and converted beatmaps only)
```

File: wiki/Interface/en.md [hunk 5]
```diff
@@ -182,7 +182,7 @@ You may also enter a beatmap or beatmapset ID in your search to get a single res
 
  A variety of things can appear in this space:
 
-- A "Not Submitted" box denotes a beatmap that has not been uploaded to the osu! site using the Beatmap Submission System.
+- A "Not Submitted" box denotes a beatmap that has not been uploaded to the osu! site using the Beatmap Submission System or deleted by mapper.
```
Comment (TPGPL) [inline wiki/Interface/en.md:185]:
```suggestion
- A "Not Submitted" box denotes a beatmap that has not been uploaded to the osu! site using the Beatmap Submission System or was deleted by the mapper.
```

File: wiki/Interface/en.md [hunk 6]
```diff
@@ -217,7 +217,7 @@ While you are on the Quick Web Access Screen, you cannot access the Chat and Ext
 The song list displays all available beatmaps. Different beatmaps may have different coloured boxes:
 
 - **Pink**: This beatmap has not been played yet.
-- **Orange**: This beatmap has been played, or all beatmaps in this mapset have been played.
+- **Orange**: This beatmap has been played, or one of the beatmap in this mapset have been played.
```
Comment (TPGPL) [inline wiki/Interface/en.md:220]:
```suggestion
- **Orange**: At least one beatmap from the beatmapset has been completed.
```

File: wiki/Interface/en.md [hunk 7]
```diff
@@ -243,7 +243,7 @@ Click the `Mods` button or press `F1` to open the **[Mod Selection Screen](/wiki
...
 
-Place your mouse on a mod's icon to see a short description of its effect. Click on an icon to select or deselect that mod. Some mods, like Double Time, have multiple variations; click on the mod again to cycle through. The score multiplier value displays the combined effect the multipliers of the mod(s) of you have selected will have on your score. Click "Reset all mods" or press 1 to deselect all currently selected mods. Click `Close` or press `2` or `Esc` to return to the Song Selection Screen.
+Place your mouse on a mod's icon to see a short description of its effect. Click on an icon to select or deselect that mod. Some mods, like Double Time, have multiple variations; click on the mod again to cycle through. The score multiplier value displays the combined effect the multipliers of the mod(s) of you have selected will have on your score. Click "Reset all mods" or press `1` to deselect all currently selected mods. Click `Close` or press `2` or `Esc` to return to the Song Selection Screen.
```
Comment (TPGPL) [inline wiki/Interface/en.md:246]:
```suggestion
Place your mouse on a mod's icon to see a short description of its effect. Click on an icon to select or deselect that mod. Some mods, like Double Time, have multiple variations; click on the mod again to cycle through. The score multiplier value displays the combined effect the multipliers of the mod(s) of you have selected will have on your score. Click `Reset all mods` or press `1` to deselect all currently selected mods. Click `Close` or press `2` or `Esc` to return to the Song Selection Screen.
```

File: wiki/Interface/en.md [hunk 8]
```diff
@@ -256,7 +256,7 @@ Click the `Random` button or press `F2` to have the game **randomly scroll throu
 Click the `Beatmap Options` button or press `F3` or right-click from mouse to call up the **Beatmap Options Menu for options on the currently selected beatmap**.
 
 - Press `1` or click the `Manage Collections` button to bring up the Collections screen - here, you can manage pre-existing collections, as well as add or remove the currently selected beatmap or mapset to or from a collection.
-- Press `2` or click `Delete...` to delete the \[1\] currently selected mapset, \[2\] delete the currently selected beatmap, or \[3\] delete all beatmaps. The last option is a joke; clicking it will cause the game to notify you that you can't delete all of your beatmaps at once.
+- Press `2` or click `Delete...` to delete the \[1\] currently selected mapset, \[2\] delete the currently selected beatmap, or \[3\] delete all VISIBLE beatmaps.
```
Comment (TPGPL) [inline wiki/Interface/en.md:259]:
```suggestion
- Press `2` or click `Delete...` to delete the \[1\] currently selected beatmapset, \[2\] delete the currently selected beatmap, or \[3\] delete **all visible beatmaps**.
```

File: wiki/Interface/en.md [hunk 9]
```diff
@@ -307,7 +307,7 @@ This is your online leaderboard. You can go here by scrolling down from the resu
    2. `Accuracy`: Your [average accuracy](/wiki/Accuracy#accuracy) over all maps you have played.
    3. `Max Combo`: The longest combo over all maps you have played.
    4. [`Ranked Score`](/wiki/Score#ranked-score): The number of points earned from all ranked maps that you have ever played, with every map being counted exactly once.
-   5. [`Total Score`](/wiki/Score#total-score): Same as ranked score, but it takes into account all types of ranking (Loved, Qualified, Ranked, Approved), underplayed and failed maps.
+   5. [`Total Score`](/wiki/Score#total-score): Same as ranked score, but it takes into account all beatmap that uploaded to osu!site (Except for not uploading or updating), underplayed and failed maps.
```
Comment (TPGPL) [inline wiki/Interface/en.md:310]:
```suggestion
   5. [`Total Score`](/wiki/Score#total-score): Same as ranked score, but it takes into account all beatmaps available on the osu! website, and also underplayed or failed beatmaps.
```

File: wiki/Interface/zh.md [hunk 10]
```diff
@@ -2,46 +2,52 @@
 outdated: true
 ---
 
-# Interface (中文)
+# 界面
```
Comment (TPGPL) [inline wiki/Interface/zh.md:5]: delete the outdated mark at the top if the translation fully matches the English article, and is up-to-date

**Review by TPGPL — CHANGES_REQUESTED** [review]

File: wiki/Interface/en.md (hunk 8 above)
Comment (cdwcgt) [reply wiki/Interface/en.md:259]: I think visible needs to be capitalized like game.

File: wiki/Interface/zh.md (hunk 10 above)
Comment (cdwcgt) [reply wiki/Interface/zh.md:5]: ok

File: wiki/Interface/en.md [hunk 11]
```diff
@@ -154,7 +154,7 @@ Supported filters:
 - `cs`: Circle Size
 - `od`: Overall Difficulty
 - `hp`: HP Drain Rate
-- `keys`: Number of keys (osu!mania maps only)
+- `keys`: Number of keys (osu!mania and converted beatmaps only)
```
Comment (PercyDan54) [inline wiki/Interface/en.md:157]:
```suggestion
- `keys`: Number of keys (converted osu!mania beatmaps only)
```
XK mod is only valid to converted mania maps

File: wiki/Interface/en.md [hunk 12]
```diff
@@ -307,7 +307,7 @@ This is your online leaderboard. You can go here by scrolling down from the resu
    2. `Accuracy`: Your [average accuracy](/wiki/Accuracy#accuracy) over all maps you have played.
```
Comment (PercyDan54) [inline wiki/Interface/en.md:307]:
```suggestion
   2. `Accuracy`: Your average [accuracy](/wiki/Accuracy#accuracy) over all maps you have played.
```

File: wiki/Interface/zh.md [hunk 13]
```diff
@@ -1,47 +1,49 @@
...
-- ［11］用户信息旁边的是地图总数，运行时间，以及电脑时显示间。
-- ［12］用户面板，点击显示用户选项。
+- \[2\] 点击 Play （`P`） 或者 osu! 标志进入次级选择界面。
+- \[3\] 点击 Edit （`E`） 进入地图编辑器的歌曲选择界面。
+- \[4\] 点击 Option （`O`） 进入设置界面。
+- \[5\] 点击 Exit （`Esc`） 退出 osu!
+- \[6\] 在主菜单下方会有随机的游戏提示。
+- \[7\] 左下角是 osu! 官网的链接，以及版权信息。
+- \[8\] 与[Bancho](/wiki/Glossary#bancho)的连接状态，在这张图总没有显示，但是连接的结果应该像一个链条。
+- \[9\] 右下角的是扩展[聊天窗口](/wiki/Chat_Console)的控制开关 (叫做"玩家列表") 和常规聊天窗口 (`F9` & `F8`, respectively).
```
Comment (PercyDan54) [inline wiki/Interface/zh.md:18]:
```suggestion
- \[9\] 右下角的是扩展[聊天窗口](/wiki/Chat_Console)的控制开关 (叫做"玩家列表") 和常规聊天窗口 （分别对应 `F9` 和 `F8`）。
```

File: wiki/Interface/zh.md [hunk 14]
```diff
@@ -51,10 +53,260 @@ outdated: true
 
 ![](img/Multi-play!_Host.jpg)
 
+1. 你在房间中的身份。这也会显示在你名字前面。
+2. 你的个人信息
+3. 音乐控制器
+4. 玩家列表 - 显示玩家的名字，他们的身份（房主或玩家），他们选择的mods（参见 #7） ,他们的osu!排名和他们的队伍（如果可用）。
```
Comment (PercyDan54) [inline wiki/Interface/zh.md:59]:
```suggestion
4. 玩家列表 - 显示玩家的名字，他们的身份（房主或玩家），他们选择的[mods](/wiki/Game_modifier) （参见 #7），他们的osu!排名和他们的队伍（如果可用）。
```

File: wiki/Interface/zh.md [hunk 15]
```diff
@@ -51,10 +53,260 @@ outdated: true
...
+5. 房间的名字和密码设定。
+6. 选中的谱面。应与在单人模式中相同的样式显示。
+7. 选中的mods （参见 #12） ，以及设置，叫做“Free Mods”的选项决定玩家是否能选择自己想使用的mods。如果启用，他们可以选择任意组合的mods*除了影响速度的模组像Double Time*。如果禁用，则由房主来决定使用什么模组，房主可以使用影响速度的mods不论Free Mods是否开启。
```
Comment (PercyDan54) [inline wiki/Interface/zh.md:62]:
```suggestion
7. 选中的[mods](/wiki/Game_modifier) （参见 #12），以及设置，叫做“Free Mods”的选项决定玩家是否能选择自己想使用的mods。如果启用，他们可以选择任意组合的mods*除了影响速度的模组，如[Double Time](/wiki/Game_modifier/Double_Time)。如果禁用，则由房主来决定使用什么模组，房主可以使用影响速度的mods不论Free Mods是否开启。
```
在en.md 最好也做出对应更改

File: wiki/Interface/en.md (hunk 12 above)
Comment (cdwcgt) [reply wiki/Interface/en.md:307]: This is what the old version looks like. I haven't changed it, but I will edit it

File: wiki/Interface/zh.md (hunk 13 above)
Comment (cdwcgt) [reply wiki/Interface/zh.md:18]: 这里眼瞎漏了（

File: wiki/Interface/zh.md (hunk 14 above)
Comment (cdwcgt) [reply wiki/Interface/zh.md:59]: 好的，英语版本我也改一下

File: wiki/Interface/zh.md [hunk 16]
```diff
@@ -1,47 +1,49 @@
...
 ## 游戏界面
 
-![](img/Introscreen2.jpg "Play Menu")
+![](img/Introscreen2.jpg "游戏界面")
 
-- 点击 Solo（P）以进行单机游戏
-- 点击 Multi（M）以进行多人联机
-- 点击 Back 以返回主菜单
+- 点击 `Solo`（`P`）以进行单机游戏
+- 点击 `Multi`（`M`）以进行多人联机
```
Comment (TPGPL) [inline wiki/Interface/zh.md:45]: missing one sentence, original line:
```md
- Click `Multi` (`M`) to play with other people. You will be directed to the [Multi](/wiki/Multi) Lobby (see below).
```

File: wiki/Interface/zh.md [hunk 17]
```diff
@@ -51,10 +53,260 @@ outdated: true
 
 ![](img/Multi-play!_Host.jpg)
```
Comment (TPGPL) [inline wiki/Interface/zh.md:54]: missing tooltip for this and the image above,  original:
```md
![](img/Multi-Play-2016.png "Multi Lobby")

![](img/Multi-play!_Host.jpg "Multi Host")
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

File: wiki/Interface/zh.md (hunk 16 above)
Comment (cdwcgt) [reply wiki/Interface/zh.md:45]: Sorry for forget that,
This sentence use the outdated text directly and forget to check.

**Review by TPGPL — APPROVED** [review]

**Pull request merged**